Bot/Discord.Bot.Extensions/Mafia/Models/Game.cs

```diff
@@ -42,7 +42,7 @@ public static Game CreateGame(List<IUser> mentions, int numMafias, GameMode mode
             if (mentions == null)
                 throw new ArgumentNullException(nameof(mentions));
 
-            if (mentions.Where(u => u.IsBot || u.IsWebhook).Count() > 0)
```

Comment:
These are all the Visual Studio style warnings/performance optimizations from when I touched the mafia command.

Reply:
No behavioral changes here.